playbooks/common/openshift-cluster/update_repos_and_packages.yml

```diff
@@ -9,4 +9,6 @@
           lookup('oo_option', 'rhel_skip_subscription') | default(rhsub_skip, True) |
             default('no', True) | lower in ['no', 'false']
   - openshift_repos
-  - os_update_latest
+  - role: os_update_latest
+    when: lookup('oo_option', 'skip_pkg_update') | default('no', True) |
```

Comment:
This should probably be something like:
```
when: openshift_skip_pkg_update | default(lookup('oo_option', 'skip_pkg_update') | default(False, True) | bool
```

That way users using the byo playbooks can use openshift_skip_pkg_update instead of cli_skip_pkg_update while retaining support for the bin/cluster options and support all of the true/false values supported by ansilbe (bool filter).